src/services/get-road-image.service.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Theme, makeStyles } from '@material-ui/core';
+import road90 from '../assets/road90.svg.png';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: theme.spacing(7),
```

Comment:
use static, `px` size - spacing should be used for theme-based attribute, like padding, marging etc.